### DISCUSSION TITLE
Add Support for HTML in Email Templates

### DISCUSSION THREAD

**Discussion by ShlomiPorush** [docusealco/docuseal#183] [Ideas]
Hi. It would be very nice if you could add support for HTML and not just plain text to any of the emails sent out.
(I think it is quite necessary at least in the Pro plan)

**Comment by omohokcoj** [comment]
@ShlomiPorush moving this into discussion. Personally i'm agains adding HTML to emails since with more email customization options users will start misusing it and the quality of emails will suffer (higher spam rate and unhappy signers). 
Still i'm open to have this feature available for Pro users if their use case really requires HTML

**Reply by ShlomiPorush** [reply]
Hi, thanks for the quick reply.
In our usecase we send RTL (hebrew) emails. 
so, the emails dont look very nice and professional.

**Reply by omohokcoj** [reply]
@ShlomiPorush we added dir="auto" to emails layout so RTL languages text should align correctly:
https://github.com/docusealco/docuseal/blob/master/app/views/layouts/mailer.html.erb#L9

Maybe it depends on the email client (i suspect Outlook doesn't support dir="auto")- can you please provide a screenshot of the email without working RTL?

Anyway, we can just fix RTL emails without adding support for HTML templates in emails

**Reply by MosesCom** [reply]
True. i did test it in outlook - the email is LTR (just checked with gmail = works)
I definitely agree that html is not needed for this one in our usecase.

**Reply by omohokcoj** [reply]
@ShlomiPorush thanks for reporting this - we will fix RTL in outlook in the next release

**Comment by benjpujol** [comment]
Hello @omohokcoj,
I am a Pro User and I think supporting HTML would be useful.
Even a little formatting of the email (bold, link formatting) could increase conversion rates and perceived quality of Docuseal by end users.
I understand the risk of affecting the deliverability due to HTML, but that remains our problem not yours.
Let me know what you think !

**Reply by omohokcoj** [reply]
Hi @benjpujol HTML emails are supported in Pro on-premises 1.4.7.
You just need to pass `<html></html>` via API or in the /settings/personalization
Please let me know if everything works as expected.
FYI @ShlomiPorush

**Reply by benjpujol** [reply]
Thank you for the quick response. Do you plan to support it on the cloud plan as well ?

**Reply by omohokcoj** [reply]
@benjpujol in the Cloud the requirement for HTML emails is to have your Gmail/Outlook connected so that the emails are sent from your email address. Can you please email support@docuseal.co with your account name/email so that we can see if this feature can be turned on for you?
